tests/system/feature_store/test_feature_store.py

```diff
@@ -3705,6 +3705,51 @@ def test_relation_asof_join(self, with_indexes):
         )
         assert_frame_equal(join_employee_department, resp_1.to_dataframe())
 
+    @pytest.mark.parametrize("with_indexes", [True, False])
+    def test_pandas_ingest_from_parquet(self, with_indexes):
+        data = dict(
+            {
+                "vakyvqqs": [-1695310155621342, 9916582819298152, 4545678706539994],
+                "ckasipbb": [
+                    0.7052167561922659,
+                    0.9029099770737461,
+                    0.7156361441244429,
+                ],
+                "enfmtxfg": [
+                    "hwroaomkupgvwmgm",
+                    "ytxmgxtgjzhyacur",
+                    "abeamnfuyrvtzwqk",
+                ],
+                "hmwaebdl": [-5274748451575421, 754465957511269, 6582195755482209],
+                "ihoubacn": [
+                    0.6705152809781331,
+                    0.09957097874816279,
+                    0.815459038897896,
+                ],
+            }
+        )
+        orig_df = pd.DataFrame(data)
+        if with_indexes:
+            orig_df.set_index(["enfmtxfg", "hmwaebdl"], inplace=True)
+        orig_df.to_parquet("/v3io/bigdata/trfsinojud.parquet")
```

Comment:
this will not work when running the test locally (ie. not from inside the jupyter pod) as there is no fuse mount.
see test_get_offline_features_with_or_without_indexes for reference. Notice that saving the parquet inside the project makes deleting it redundent, as the project files are deleted at the end of the test.